logserver/pom.xml

```diff
@@ -47,11 +47,15 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.30</version>
+        </dependency>
```

Comment:
is this dependency only applied to the test? If it is only for the test, it should be only applied to the test.
From my recent experience, unnecessary dependency might cause problem.

Reply:
Yes, see this commit https://github.com/Alluxio/alluxio/commit/124be7dc788a4cfc52d285b1f2f719a05e2bccc5